contracts/optimistic-ethereum/OVM/chain/OVM_CanonicalTransactionChain.sol

```diff
@@ -128,15 +133,15 @@ contract OVM_CanonicalTransactionChain is iOVM_CanonicalTransactionChain, Lib_Ad
     /**
      * @inheritdoc iOVM_CanonicalTransactionChain
      */
-    function getNextQueueIndex()
+    function getNextPendingQueueIndex()
```

Comment:
@tynes this will need updating in some other processes, but @karlfloersch and I thought worth changing